Bio/Align/__init__.py

```diff
@@ -953,6 +953,22 @@ def substitutions(self):
 
         return m
 
+    @property
+    def alignment(self):
+        """Return an Alignment object based on the MultipleSeqAlignment object."""
+        records = [copy.copy(record) for record in self._records]
```

Comment:
Taking a copy worried me, but given you're building a SeqRecord based alignment and need to remove any gaps in the sequence, probably unavoidable.

You could special case gap-less records, but that seems even more fragile.

I think this should be documented though. I can imagine use cases where for example people amend the SeqRecord annotation - which would not propogate as there are now two independent versions of each record. e.g. Add something like this to the docstring?:

``This makes a copy of each SeqRecord with a gap-less sequence. Any future changes to the original records in the MultipleSeqAlignment will not affect the new records in the Alignment.``